WPAndroid Integration.md

```diff
@@ -52,7 +52,7 @@ implementation "$rootProject.gradle.ext.gutenbergMobileBinaryPath:$rootProject.e
 There are a few different ways to do this:
 
 1. Open a new PR or push a new commit to an open PR which will be published as `<PR number>-<commit full SHA1>`
-2. Merge a PR to `develop` which will be published as `<develop>-<commit full SHA1>`
+2. Merge a PR to `trunk` which will be published as `<trunk>-<commit full SHA1>`
```

Comment:
Hi @oguzkocer, can you help me confirm this change is accurate?
I've checked a few other PRs against trunk and development in buildkite but i'm not sure if anything else needs updating. thanks!

Reply:
Yeap, that's accurate.